packages/tempus/src/Thyra_ReuseLinearOpWithSolveFactory.hpp

```diff
@@ -33,6 +33,7 @@ class ReuseLinearOpWithSolveFactory
    *
    * \param lowsf [in,persisting] The LOWSFB object that will be used to
    * create the LOWSB object for the diagonal blocks.
+   * \param prec [?] Description.
```

Comment:
Fill these out too.

Reply:
I copied these from Thyra, so the fact they are missing isn't surprising...